### DISCUSSION TITLE
Some suggestions on frequent version updating

### DISCUSSION THREAD

**Discussion by dibadibadiudiu** [tauri-apps/tauri#1790] [Ideas]
I hope that every time I run the `yarn tauri dev / build` command, will check whether the current library version is up-to-date. If not, give a tip.

In addition, add version library information to the example code, such as `tauriv1.0.0-beta-rc.1`. I often read the example code, but they can't run because of different library versions.

Thank you for the efforts of the development team. Your project is excellent! 👍

**Comment by nothingismagick** [comment]
I can see the need for this, but I think that the feature would be to have a "tooling" setting in the `tauri.conf.json` that allows you to specify. I see this potentially wreaking havoc on CI though
